config/database.yml

```diff
@@ -1,19 +1,20 @@
-base: &base
-  encoding: utf8
-  url: <%= ENV.fetch("DATABASE_URL", "trilogy://root@localhost") %>
+default: &default
+  adapter: trilogy
+  encoding: utf8mb4           # <-- forces UTF-8 everywhere
+  collation: utf8mb4_unicode_ci  # <-- portable across MySQL 5.7/8.0 and MariaDB
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS", 5) %>
+  user: root
+  password:
```

Comment:
These should either be overridable  ... or we should have some documentation in here *explicitly* pointing towards setting the `DATABASE_URL` envvar to override with local config.  Not everyone knows that's a thing.